<<<<<<<< HEAD:patch/kernel/archive/sunxi-6.15/patches.megous/arm64-dts-sun50i-a64-pinephone-Use-newer-jack-detection-impleme.patch
From 0a1d2105089307fe586728da81ffe9ff72a7c1c1 Mon Sep 17 00:00:00 2001
========
From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
>>>>>>>> 09b789500459353b662b9482ae7912879c942f1a:patch/kernel/archive/sunxi-6.18/patches.megous/pp-6.18/0022-arm64-dts-sun50i-a64-pinephone-Use-newer-jack-detect.patch
From: Ondrej Jirman <megi@xff.cz>
Date: Fri, 23 Feb 2024 01:53:14 +0100
Subject: arm64: dts: sun50i-a64-pinephone: Use newer jack detection
 implementaion

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
<<<<<<<< HEAD:patch/kernel/archive/sunxi-6.15/patches.megous/arm64-dts-sun50i-a64-pinephone-Use-newer-jack-detection-impleme.patch
index aeb282eba4b5..e7e90e5baff4 100644
========
index 111111111111..222222222222 100644
>>>>>>>> 09b789500459353b662b9482ae7912879c942f1a:patch/kernel/archive/sunxi-6.18/patches.megous/pp-6.18/0022-arm64-dts-sun50i-a64-pinephone-Use-newer-jack-detect.patch
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -476,6 +476,7 @@ keyboard-power {
 &codec {
 	pinctrl-names = "default";
 	pinctrl-0 = <&aif2_pins>, <&aif3_pins>;
+	jack-type = "headset";
 	status = "okay";
 };
 
@@ -971,7 +972,7 @@ reboot_mode: reboot-mode@4 {
 &sound {
 	status = "okay";
 	simple-audio-card,name = "PinePhone";
-	simple-audio-card,aux-devs = <&codec_analog>, <&speaker_amp>;
+	simple-audio-card,aux-devs = <&codec_analog>, <&codec>, <&speaker_amp>;
 	simple-audio-card,widgets = "Microphone", "Headset Microphone",
 				    "Microphone", "Internal Microphone",
 				    "Headphone", "Headphone Jack",
-- 
Armbian

